skyportal/onboarding.py

```diff
@@ -165,3 +165,57 @@ def setup_invited_user_permissions(strategy, uid, details, user, *args, **kwargs
 
     invitation.used = True
     DBSession().commit()
+
+
+def user_details(strategy, details, backend, uid, user=None, *args, **kwargs):
+    """Update user details using data from provider."""
+    if not user:
+        return
+    existing_user = DBSession().query(User).filter(User.oauth_uid == uid).first()
```

Comment:
What happens for users added programmatically, who do not have an oauth_uid.

In that case, existing user will probably evaluate to None?

Reply:
Yes, thanks, I will check if it's None first!

Reply:
@stefanv actually, I just looked at it and think it's right as is. If we're adding users programmatically that will auth using real OAuth, we'll need to set their oauth_uid when we create the users for it to work anyways. So this shouldn't be an issue. I just checked it with debug_login and it works as expected.